README.md

```diff
@@ -218,59 +218,21 @@ where
-Since bundles' filelists are serialized filesystem representations,
-the `wrap_datamon.sh` interface is tightly coupled to that of the self-documenting
-`datamon` binary itself.
-
-```shell
-./wrap_datamon.sh -c <coord_dir> -d <bin_cmd_I> -d <bin_cmd_J> ...
-```
-
-* `-c` the same coordination directory passed to `wrap_application.sh`
-* `-d` all parameters, exactly as passed to the datamon binary, except as a
-  single scalar (quoted) parameter, for one of the following commands
-  - `config` sets user information associated with any bundles created by the node
-  - `bundle mount` provides sources for data-science applications
-  - `bundle upload` provides sinks for data-science applications
-
-Multiple (or none) `bundle mount` and `bundle upload` commands may be specified,
-and at most one `config` command is allowed so that an example `wrap_datamon.sh`
-YAML might be
-
-```yaml
-command: ["./wrap_datamon.sh"]
-args: ["-c", "/tmp/coord", "-d", "config create --name \"Coord\" --email coord-bot@oneconcern.com", "-d", "bundle upload --path /tmp/upload --message \"result of container coordination demo\" --repo ransom-datamon-test-repo --label coordemo", "-d", "bundle mount --repo ransom-datamon-test-repo --label testlabel --mount /tmp/mount --stream"]
-```
-
-or from the shell
-
-```shell
-./wrap_datamon.sh -c /tmp/coord -d 'config create --name "Coord" --email coord-bot@oneconcern.com' -d 'bundle upload --path /tmp/upload --message "result of container coordination demo" --repo ransom-datamon-test-repo --label coordemo' -d 'bundle mount --repo ransom-datamon-test-repo --label testlabel --mount /tmp/mount --stream'
-```
-
-##### `gcr.io/onec-co/datamon-pg-sidecar` -- `wrap_datamon_pg.sh`
-
-Provides Postgres databases as bundles and vice versa.
-Since the datamon binary does not include any Postgres-specific notions,
-the UI here is more decoupled than that of `wrap_datamon.sh`.
-The UI is specified via environment variables
-such that `wrap_datamon.sh` is invoked without parameters.
-
-The script looks for precisely one `dm_pg_opts` environment variable
-specifying global options for the entire script and any number of
-`dm_pg_db_<db_id>` variables, one per database.
+Meanwhile, each sidecar's datamon-specific batteries have their corresponding
+usages.
+Despite their differences, both are parameterized via environment variables.
+[12 factor](https://www.12factor.net/config) principles are loosely part
+of the overall ideology with the strong opinion being that of the virtue of
+declarative specification.
+The environment variables for sidecars both use the same serialization
+format to represent dictionaries (aka associative arrays) with strings.
 
 ----
 
 _Aside on serialization format_
 
-Each of these environment variables each contain a serialized dictionary
-according the the following format
+Each of environment variable parsed by the sidecars contains a serialized
```

Comment:
If the dictionary is fixed list of strings, can they be their own environment variable vs packing them into the value of an environment variable.

Reply:
this is the *bridge to internal zsh variables* section of [the batch comment](https://github.com/oneconcern/datamon/pull/270#issuecomment-543291247).

there are multiple options for the bridge.  the list in the comment is probably not even exhaustive.